stellarphot/photometry/tests/test_photometry.py

```diff
@@ -39,77 +39,107 @@
-    pixel_scale=1 * u.arcsec / u.pixel,
-    max_data_value=40000 * u.adu,
-)
+@pytest.fixture
+def photometry_optional_settings():
+    pos = PhotometryOptionalSettings()
+    # This used to be the default; it has switched to exact
+    pos.method = "center"
+    return pos
+
+
+@pytest.fixture
+def fake_camera():
+    # A camera with not unreasonable settings
+    return Camera(
+        data_unit=u.adu,
+        gain=1.0 * u.electron / u.adu,
+        name="test camera",
+        read_noise=0 * u.electron,
+        dark_current=0.1 * u.electron / u.second,
+        pixel_scale=1 * u.arcsec / u.pixel,
+        max_data_value=40000 * u.adu,
+    )
+
+
+@pytest.fixture
+def zero_camera():
+    # Camera with no read noise or dark current
+    return Camera(
+        data_unit=u.adu,
+        gain=1.0 * u.electron / u.adu,
+        name="test camera",
+        read_noise=0 * u.electron,
+        dark_current=0.0 * u.electron / u.second,
+        pixel_scale=1 * u.arcsec / u.pixel,
+        max_data_value=40000 * u.adu,
+    )
+
+
+@pytest.fixture
+def fake_obs():
+    # Fake observatory location
+    return Observatory(
+        name="test observatory",
+        latitude=0 * u.deg,
+        longitude=0 * u.deg,
+        elevation=0 * u.m,
+    )
 
-# Fake observatory location
-FAKE_OBS = Observatory(
-    name="test observatory", latitude=0 * u.deg, longitude=0 * u.deg, elevation=0 * u.m
-)
 
 # The fake image used for testing
 FAKE_CCD_IMAGE = FakeCCDImage(seed=SEED)
 
+
 # Build default PhotometryOptions for the tests based on the fake image
-DEFAULT_PHOTOMETRY_APERTURES = PhotometryApertures(
-    radius=FAKE_CCD_IMAGE.sources["aperture"][0],
-    gap=FAKE_CCD_IMAGE.sources["aperture"][0],
-    annulus_width=FAKE_CCD_IMAGE.sources["aperture"][0],
-    fwhm=FAKE_CCD_IMAGE.sources["x_stddev"].mean(),
-)
+@pytest.fixture
+def photometry_apertures():
+    return PhotometryApertures(
+        radius=FAKE_CCD_IMAGE.sources["aperture"][0],
+        gap=FAKE_CCD_IMAGE.sources["aperture"][0],
+        annulus_width=FAKE_CCD_IMAGE.sources["aperture"][0],
+        fwhm=FAKE_CCD_IMAGE.sources["x_stddev"].mean(),
+    )
+
 
 # Passband map for the tests
-PASSBAND_MAP = PassbandMap(
-    name="Example Passband Map",
-    your_filter_names_to_aavso={
-        "B": "B",
-        "rp": "SR",
-    },
-)
+@pytest.fixture
+def passband_map():
+    return PassbandMap(
+        name="Example Passband Map",
+        your_filter_names_to_aavso={
+            "B": "B",
+            "rp": "SR",
+        },
+    )
 
-DEFAULT_LOGGING_SETTINGS = LoggingSettings()
 
-DEFAULT_PHOTOMETRY_SETTINGS = PhotometrySettings(
-    camera=FAKE_CAMERA,
-    observatory=FAKE_OBS,
-    photometry_apertures=DEFAULT_PHOTOMETRY_APERTURES,
-    source_location_settings=DEFAULT_SOURCE_LOCATIONS,
-    photometry_optional_settings=PHOTOMETRY_OPTIONS,
-    passband_map=PASSBAND_MAP,
-    logging_settings=DEFAULT_LOGGING_SETTINGS,
-)
+@pytest.fixture
+def photometry_settings(
```

Comment:
I know its just a pytest fixture, but I thought it might make sense to call this `test_photometry_settings` since there are other places in the code where we have photometry settings getting set.

Reply:
How about `photometry_settings_for_test`? I'm not sure how well `pytest` will react if the name starts with `test`...

Reply:
Sounds good, I forgot pytest is going to look for particular function names.